app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -672,8 +672,8 @@ protected void initListeners() {
             if (motionEvent.getAction() == MotionEvent.ACTION_DOWN) {
                 animate(binding.touchAppendDetail, true, 250, AnimationType.ALPHA,
                         0, () ->
-                        animate(binding.touchAppendDetail, false, 1500,
-                                AnimationType.ALPHA, 1000));
+                                animate(binding.touchAppendDetail, false, 1500,
+                                        AnimationType.ALPHA, 1000));
```

Comment:
Oh, strange, this shouldn't happen. Try to apply this suggestion:
```suggestion
                        animate(binding.touchAppendDetail, false, 1500,
                                AnimationType.ALPHA, 1000));
```